benches/bench.rs

```diff
@@ -152,7 +154,8 @@ fn bench_parsing_line_number_program_opcodes(b: &mut test::Bencher) {
     let debug_line = DebugLine::<LittleEndian>::new(&debug_line);
 
     b.iter(|| {
-        let program = debug_line.program(OFFSET, ADDRESS_SIZE, None, None)
+        let program = debug_line
+            .program(OFFSET, ADDRESS_SIZE, None, None)
```

Comment:
This behaviour is controlled by `closure_block_indent_threshold`, which defaults to 7 lines. So closures of up to 7 lines use visual, and longer closures use block indent. I'd be happy with a setting that made it use the same indentation always.